code/modules/reagents/reagent_containers.dm

```diff
@@ -70,6 +70,8 @@
 	if(standard_splash_mob(user, target))
 		return
 	if(standard_pour_into(user, target))
+		if(SSvueui.get_open_uis(target)) // blame whoever decided beakers should go in afterattack()
+			SSvueui.check_uis_for_change(target)

```

Comment:
Remove if, just do check for change. You are just wasting time.

Reply:
Fair enough, wasn't sure if one was cheaper than the other.